ethers-signers/src/trezor/app.rs

```diff
@@ -73,12 +75,25 @@ impl TrezorEthereum {
     }
 
     fn check_version(version: String) -> Result<(), TrezorError> {
-        let req = semver::VersionReq::parse(FIRMWARE_MIN_VERSION)?;
         let version = semver::Version::parse(&version)?;
+        let mut min_version;
 
-        // Enforce firmware version is greater than FIRMWARE_MIN_VERSION
+        match version.major {
+            1 => {
+                min_version = FIRMWARE_1_MIN_VERSION;
+            }
+            2 => {
+                min_version = FIRMWARE_2_MIN_VERSION;
+            }
```

Comment:
```suggestion
        let min_version = match version.major {
            1 => FIRMWARE_1_MIN_VERSION,
            2 => FIRMWARE_2_MIN_VERSION,
```

Reply:
addressed in dd108acb51141e98cfc734d7ebe2e2ed2ebbcd27